Makefile

```diff
@@ -97,17 +98,46 @@ test: ## Run all unit tests
 .PHONY: build
 build: build/opengamepad-ui.x86_64 ## Build and export the project
 build/opengamepad-ui.x86_64: $(PROJECT_FILES) $(EXPORT_TEMPLATE)
+	@echo "Building OpenGamepadUI v$(OGUI_VERSION)"
 	mkdir -p build
 	$(GODOT) --headless --export-debug "Linux/X11"
 
 .PHONY: update-pack
-update-pack: build/update.pck ## Build and export update pack
+update-pack: build/update.pck ## Build and export update pack (DEPRECATED)
```

Comment:
How long are we keeping these deprecated entries in the Makefile?

Reply:
After this merges, I was going to remove them in a follow up PR.